pkg/virt/virt.go

```diff
@@ -173,17 +173,22 @@ func genContainerCgroupPath(ctype cgroups.ControlType, id string, cgroup2 bool,
 					node_hostname = strings.Split(node_hostname, ".")[0]
 
 					// from `container_name` get the name of the ceph service for which cgroup path is to be found
-					ceph_service_name := strings.Split(container_name, node_hostname)[0]
-					ceph_service_name = strings.Trim(ceph_service_name, "-")
-					ceph_service_name = strings.SplitAfter(ceph_service_name, fmt.Sprintf("ceph-%s-", ceph_fsid))[1]
```

Comment:
ceph_service_name is not used afterwards anymore, in that case it can completely go away?

Reply:
I realized using `ceph_service_name` improves code readability, so I've reintroduced the variable. In the latest changes, we obtain ceph service's name from the container name.

Reply:
This might have changed after the comment, but right now ceph_service_name is used at L191.